apps/web/tsconfig.json

```diff
@@ -21,6 +21,11 @@
       "@assets/*": ["public/assets/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    "cypress/integration/1-getting-started/partone.spec.js"
```

Comment:
Is this complete?

Reply:
I think this is auto generated, I'll clear it out.